tests/python/unittest/test_image.py

```diff
@@ -155,6 +155,17 @@ def test_imageiter(self):
 
 
     def test_augmenters(self):
```

Comment:
Please add the with-seed annotation

Reply:
Could you please point to an example so I can add it?

Reply:
Nevermind, found it :)